source/frontend/src/customAxios.ts

```diff
@@ -42,6 +42,7 @@ export const CustomAxios = ({
     baseURL,
     headers: {
       'Access-Control-Allow-Origin': '*',
+      'Cache-Control': process.env.NODE_ENV === 'development' ? 'no-cache' : '',
```

Comment:
In development, don't cache anything as we may be changing lookup codes around a bunch.